src/internal.c

```diff
@@ -15977,19 +16064,7 @@ static int DoAlert(WOLFSSL* ssl, byte* input, word32* inOutIdx, int* type,
         return ALERT_COUNT_E;
     }
 
-    WOLFSSL_MSG("Got alert");
-    if (*type == close_notify) {
-        WOLFSSL_MSG("\tclose notify");
-        ssl->options.closeNotify = 1;
```

Comment:
Woops, this shouldn't have been removed. Meant to just remove the logging bits that are now handled by `LogAlert`. Will fix.